themes/Flex/templates/base.html

```diff
@@ -148,18 +148,18 @@ <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
 
     {% if MAIN_MENU %}
     <nav>
-      <a href="{{ SITEURL }}">{{ _('Home') }}</a>
+      <a href="{{ SITEURL }}" aria-label="Home page">{{ _('Home') }}</a>
```

Comment:
You don't need the `aria-label` here because there's already text content in the link

Reply:
Fixed!